oeps/oep-2.rst

```diff
@@ -0,0 +1,101 @@
+===================================
```

Comment:
Can we make this be a different PR so we can finish OEP-1 first? The concerns and discussion will likely be very different.